src/main/resources/data/minecolonies/tags/items/stonemason_product.json

```diff
@@ -13,6 +13,8 @@
     "minecraft:flower_pot",
     "#forge:sandstone",
     "minecraft:chiseled_quartz_block",
-    "minecraft:quartz_pillar"
+    "minecraft:quartz_pillar",
+    "minecraft:quartz_block",
```

Comment:
No, this is already handled by the mechanic.

Reply:
Why can the stonemason craft all other quartz blocks then?

Reply:
Because at that point it's stone? :) Most ALL of the "block" variant of things is a mechanic thing, including iron and gold blocks.

Reply:
Hmm, I've always considered quartz blocks stone and not storage blocks. Opinions?

Reply:
Joining quartz into quartz blocks is fine for mechanic for me. It's kinda a special nether block and not an easy mason job to produce the block